build.sh

```diff
@@ -4,12 +4,9 @@
 # and builds AppImage
 #
 
-small_FLAGS="-Os -ffunction-sections -fdata-sections"
-small_LDFLAGS="-s -Wl,--gc-sections"
-CC="cc -O2 -Wall -Wno-deprecated-declarations -Wno-unused-result"
 
 STRIP="strip"
-INSTALL_DEPENDENCIES=1
+INSTALL_DEPENDENCIES=0
```

Comment:
Why?

Reply:
The dependencies are all there in the containers we use. Installing means having to wait for unnecessary installations to be triggered, which will return error codes.

Reply:
I don't think it's a good idea to call that by default when just trying to build the repository, especially when using non-throwaway systems like the majority of developers have. I'd keep it opt-in instead of opt-out. But if you insist, we make it opt-out again, and deactivate it on Travis.